src/eko/io/struct.py

```diff
@@ -627,6 +627,11 @@ def mu2grid(self) -> List[SquaredScale]:
         """Provide the list of :math:`Q^2` as an array."""
         return [mu2 for mu2, _ in self._operators]
 
+    @property
+    def epgrid(self) -> List[EPoint]:
```

Comment:
I think `mu2grid` should be removed in favor of this function (likewise for `mu20`), but think this is a massive change I didn't dare do it straight away ...

Reply:
Indeed, I was worried of dropping `mu2grid` immediately...

I started calling in some places `evolgrid` (sounds better than `epgrid`, that is more a code name), but here I avoided just because you could use `list(self)` for this, leveraging the definition of `__iter__` (and I believe we should do also here, if you want to explicitly expose this property, don't you think so?).

Reply:
adjusted in https://github.com/NNPDF/eko/pull/242/commits/e91e526183efad77be23790b7b085fb34f52316d - okay, but if you agree then I start to propagate it around